cfme/services/myservice/ui.py

```diff
@@ -144,7 +144,7 @@ class provisioning(View):  # noqa
                       'normalize-space(.)="Plays"]]')
         details = SummaryTable(title='Details')
         credentials = SummaryTable(title='Credentials')
-        standart_output = Text('.//div[@id="provisioning"]//pre')
+        standart_output = Text('.//div[@id="provisioning"]/ansible-raw-stdout')
```

Comment:
This doesn't break anything else, does it?

Reply:
At this point we're only supporting 5.11 and upstream with the master branch, so I'm not concerned with the potential for this to break the locator in older versions.